src/main.rs

```diff
@@ -0,0 +1,11 @@
+use slog::{o, slog_debug, slog_error, slog_info, slog_trace, slog_warn};
+use slog_scope::{debug, error, info, trace, warn};
```

Comment:
@Kirguir this would be uncomfortable to import all the time. For ergonomics, you can re-export these in a separate module and use it as `use module::*`.

Reply:
@tyranron with prelude it's cool